api-service/auth/simple_auth.go

```diff
@@ -104,7 +108,10 @@ func (ap *BasicAuthenticationProvider) GetToken(w http.ResponseWriter, r *http.R
 	}
 
 	w.WriteHeader(http.StatusOK)
```

Comment:
Would you mind to put the WriteHeader after the Write?
Nothing wrong will never happen with the Write, but if something happen the response will be already OK because of this.